recipes/default.rb

```diff
@@ -26,6 +26,8 @@
 end
 
 vault_binary = File.join(service.install_path, 'vault', 'current', 'vault')
+
+package 'libcap2-bin'
```

Comment:
I'm not sure on what platform this package is required? Just based on the library name I assume the resource below this is affected and we probably want to hoist the guards for windows/mac to encompass that.